types/round/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for round 2.0
+// Project: https://github.com/bendrucker/round.js#readme
+// Definitions by: Rajas Paranjpe <https://github.com/ChocolateLoverRaj>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace round;
```

Comment:
I think this package has only CommonJS bits published and available for consumption:
https://unpkg.com/browse/round@2.0.1/
Why do you think this is UMD?

Reply:
I thought it was umd because it adds a `round` global variable when used as a `<script>`, but now I know that It's not because it does `exports = module.exports = round`, which won't work in browser. I will remove the `export as namespace` part.